samples/treectrl/treetest.cpp

```diff
@@ -979,6 +979,8 @@ void MyTreeCtrl::CreateImages(int size)
     // should correspond to TreeCtrlIcon_xxx enum
     wxIcon icons[5];
 
+    const auto scaleFac = this->GetDPIScaleFactor();
```

Comment:
Just not to forget to do it: changes to this file should be reverted, they were useful for testing but not needed permanently.